test/libsolidity/smtCheckerTests/types/address_call.sol

```diff
@@ -20,3 +20,6 @@ contract C
 // ----
 // Warning: (224-240): Unused local variable.
 // Warning: (260-275): Assertion violation happens here
+// Warning: (279-293): Assertion violation happens here
```

Comment:
These are added because of the same reason as above?

Reply:
Exactly